.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "PHP preparations",
```

Comment:
Does this name represent the stage of the build or the name of the project? If its the latter, we should probably rename to `open-runtimes/proxy`

Reply:
Specs says `A name for the dev container displayed in the UI.`.
Ill rename to open-runtimes/proxy